testcontainers/core/container.py

```diff
@@ -38,23 +38,20 @@ def with_kargs(self, **kargs) -> 'DockerContainer':
         return self
 
     def start(self):
-        print("")
-        print("{} {}".format(crayons.yellow("Pulling image"),
-                             crayons.red(self.image)))
-        with blindspin.spinner():
-            docker_client = self.get_docker_client()
-            self._container = docker_client.run(self.image,
-                                                command=self._command,
-                                                detach=True,
-                                                environment=self.env,
-                                                ports=self.ports,
-                                                name=self._name,
-                                                volumes=self.volumes,
-                                                **self._kargs
-                                                )
-        print("")
-        print("Container started: ",
-              crayons.yellow(self._container.short_id, bold=True))
+        logger.info("")
+        logger.info("{} {}".format("Pulling image", self.image))
```

Comment:
Let's use the [standard approach](https://docs.python.org/3/howto/logging.html#logging-variable-data) for parameter substitution in logs throughout.